package.json

```diff
@@ -30,7 +30,10 @@
     "express": "^4.16.4",
     "get-folder-size": "^2.0.0",
     "ipfs": "^0.33.1",
-    "orbit-db": "^0.19.9",
+    "js-sha256": "^0.9.0",
+    "keccak": "^2.0.0",
+    "multihashes": "^0.4.14",
+    "orbit-db": "git://github.com/orbitdb/orbit-db.git#941c078",
```

Comment:
I think we are going with the version specified in this PR: https://github.com/3box/3box-pinning-server/pull/107

Reply:
Done!